.github/workflows/lint_pull_request.yml

```
@@ -6,29 +6,29 @@ on:
 
 jobs:
   clang-tidy-review:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

We can use Ubuntu latest here to reduce maintenance effort
```suggestion
    runs-on: ubuntu-latest
```